getCli.sh

```diff
@@ -3,26 +3,34 @@
 CLI_OS="na"
 CLI_UNAME="na"
 
+if [ $# -eq 0 ]
+  then
+	VERSION=\$latest
+	echo "Using latest version of jfrog cli"
+  else
+	VERSION=$1
```

Comment:
To be consistent, let's add another echo here prompting:
**echo "Downloading version $1 of JFrog CLI..."**